<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>xyzP2PWidget</class>
 <widget class="QWidget" name="xyzP2PWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>355</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>XYZP2PWidget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QGroupBox" name="controllBox">
     <property name="title">
      <string>Control</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>2</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="axisXLabel">
          <property name="text">
           <string>X:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="xPosSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="xPosSpinBox">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="axisXCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="axisYLabel">
          <property name="text">
           <string>Y:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="yPosSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="yPosSpinBox">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="axisYCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="axisZLabel_4">
          <property name="text">
           <string>Z:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="zPosSlider_2">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="zPosSpinBox_2">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="axisZCheckBox_2">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <widget class="QLabel" name="axisRLabel">
          <property name="text">
           <string>R:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="rPosSlider">
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="rPosSpinBox">
          <property name="suffix">
           <string>deg</string>
          </property>
          <property name="minimum">
           <number>-180</number>
          </property>
          <property name="maximum">
           <number>180</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="axisRCheckBox">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>7</number>
        </property>
        <item>
         <widget class="QSlider" name="refVelSlider">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="pageStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="invertedAppearance">
           <bool>false</bool>
          </property>
          <property name="invertedControls">
           <bool>false</bool>
          </property>
          <property name="tickPosition">
           <enum>QSlider::NoTicks</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="refVelSpinBox">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="suffix">
           <string>deg/s</string>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>18</number>
          </property>
          <property name="singleStep">
           <number>1</number>
          </property>
          <property name="value">
           <number>1</number>
          </property>
          <property name="displayIntegerBase">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="startBtn">
          <property name="minimumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="icons.qrc">
            <normaloff>:/icons/big_starticon.png</normaloff>
            <normalon>:/icons/big_stopicon.png</normalon>:/icons/big_starticon.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="statusBox">
     <property name="title">
      <string>Status</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <property name="spacing">
       <number>2</number>
      </property>
      <property name="leftMargin">
       <number>2</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>2</number>
      </property>
      <property name="bottomMargin">
       <number>2</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="axisXLabel_2">
          <property name="text">
           <string>AxisX:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="xVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="axisYLabel_2">
          <property name="text">
           <string>AxisY:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="yPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_3">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="yVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_3">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="axisZLabel_2">
          <property name="text">
           <string>AxisZ:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_2">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="zVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_2">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="axisRLabel_2">
          <property name="text">
           <string>AxisR:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="rPosLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="posUnit_5">
          <property name="text">
           <string>deg</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLCDNumber" name="rVelLcdNum"/>
        </item>
        <item>
         <widget class="QLabel" name="velUnit_5">
          <property name="text">
           <string>deg/s</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>refVelSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refVelSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>196</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>274</x>
     <y>177</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>refVelSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>refVelSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>280</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>243</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>xPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>xPosSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>244</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>43</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yPosSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>173</x>
     <y>83</y>
    </hint>
    <hint type="destinationlabel">
     <x>275</x>
     <y>79</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yPosSpinBox</sender>
   <signal>valueChanged(int)</signal>
   <receiver>yPosSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>291</x>
     <y>88</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>90</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zPosSlider_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zPosSpinBox_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>201</x>
     <y>119</y>
    </hint>
    <hint type="destinationlabel">
     <x>273</x>
     <y>112</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>zPosSpinBox_2</sender>
   <signal>valueChanged(int)</signal>
   <receiver>zPosSlider_2</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>274</x>
     <y>120</y>
    </hint>
    <hint type="destinationlabel">
     <x>245</x>
     <y>126</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rPosSpinBox</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>142</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>rPosSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>rPosSlider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>148</y>
    </hint>
    <hint type="destinationlabel">
     <x>260</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
